File: src/main/java/com/example/message/domain/FirebaseAuthToken.java
package com.example.message.domain;

import lombok.Getter;
import lombok.NoArgsConstructor;
import lombok.Setter;

import javax.persistence.*;

@Entity(name = "FirebaseAuthUser")
@Getter
@NoArgsConstructor
public class FirebaseAuthToken {

    public FirebaseAuthToken(String firebaseAuthToken) {
        this.firebaseAuthToken = firebaseAuthToken;
    }

    @Id
    @Column(name = "FIREBASE_AUTH_TOKEN_ID")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private long id;

    @Setter
    @Column(length = 1000)
    private String firebaseAuthToken;
}

File: src/main/java/com/example/message/domain/FirebaseAuthToken.kt
package com.example.message.domain

import javax.persistence.*

@Entity(name = "FirebaseAuthUser")
class FirebaseAuthToken(

        @Id
        @Column(name = "FIREBASE_AUTH_TOKEN_ID")
        @GeneratedValue(strategy = GenerationType.IDENTITY)
        val id: Long = -1,

        @Column(length = 1000)
        var firebaseAuthToken: String
)